.github/workflows/merged.yml

```
@@ -0,0 +1,41 @@
+        with:
+          path: .nx
+          key: ${{ github.event.pull_request.head.ref }}-NX-${{ runner.os }}-${{ matrix.node-version }}-${{ github.event.pull_request.head.sha }}
+
+      - if: ${{ steps.latest-nx-cache.outputs.cache-hit == 'true' }}
+        name: 更新公共 NX 缓存
+        uses: actions/upload-artifact@v4
+        with:
+          path: .nx
+          name: NX-${{ runner.os }}-${{ matrix.node-version }}
+          overwrite: true
+          include-hidden-files: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)